packages/imperative/CHANGELOG.md

```diff
@@ -2,9 +2,15 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Resolved bug that resulted in each plugin to have identical public registries regardless of actual installation location/reference
```

Comment:
- Plug-in is hyphenated per the Zowe doc style
- Should end w/ a period
- Please include an issue or PR link to this changelog item